core/src/main/java/com/linecorp/armeria/server/cors/AbstractCorsPolicyBuilder.java

```diff
@@ -271,6 +275,19 @@ public AbstractCorsPolicyBuilder allowRequestMethods(Iterable<HttpMethod> method
         return this;
     }
 
+    /**
+     * Enables to allow all HTTP headers.
+     *
+     * <p>The server will set the CORS {@code "Access-Control-Allow-Headers"} to be as same as the CORS
+     * {@code "Access-Control-Request-Headers"} header in the request.
+     *
+     * @return {@code this} to support method chaining.
+     */
+    public AbstractCorsPolicyBuilder allowAllRequestHeaders() {
```

Comment:
Once we change the default value of this property to `true` in 2.0, we will have the method that accepts a boolean parameter only. How about adding one here to be future-proof?